src/main/java/frc/robot/Constants.java

```diff
@@ -16,6 +16,9 @@
   public static final int kPCMId = 1;
   public static final int kShifterSolenoidId = 7;
 
+  // PWM
+  public static final int cameraRingId = 0;
```

Comment:
Should be sCameraRingId because it's a static variable

Reply:
I'm pretty sure this is also a constant, so resolved to "kCameraRingID"